specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2019-03-01/AlertsManagement.json

```diff
@@ -325,204 +325,6 @@
           }
         }
       }
-    },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/smartGroups": {
```

Comment:
>subscriptions [](start = 6, length = 13)

just making sure - in our ARM manifest we also support two more resources for this API version:
smartDetectorAlertRules
alertsMetaData

Are they defined in another file?